### ISSUE TITLE
Unable to delete user in platform manager

### ISSUE THREAD

**Issue by tatarincev** [VirtoCommerce/vc-platform#1203]
https://files.gitter.im/VirtoCommerce/vc-platform/owZl/blob
![image](https://user-images.githubusercontent.com/7566324/32325571-25b7beba-bfd8-11e7-8962-ee8751f0f6ab.png)

**Comment by Skrypt** [comment]
I think the missing method is : 

```
        /// <summary>
        /// Delete users by name
        /// </summary>
        /// <param name="names">An array of user names.</param>
        [HttpDelete]
        [Route("users")]
        [ResponseType(typeof(void))]
        [CheckPermission(Permission = PredefinedPermissions.SecurityDelete)]
        public async Task<IHttpActionResult> DeleteAsync([FromUri] string[] names)
        {
            EnsureThatUsersEditable(names);

            await _securityService.DeleteAsync(names);
            return StatusCode(HttpStatusCode.NoContent);
        }
```

**Comment by asvishnyakov** [comment]
Fixed in 8dcc3394dcf4d70b38e01a5b3019b84a126e84dc

**Comment by tatarincev** [comment]
Released https://github.com/VirtoCommerce/vc-platform/releases/tag/v2.13.18
